### PR TITLE
Split p2p by moving the reports routines

### PR DESCRIPTION
**Pull request by aquintanas**
This PR splits the `p2p` routine by defining the reports library. In this library, each report has its python routine.

### CODE REVIEW

File: test/BUILD [hunk 1]
```diff
@@ -17,3 +17,19 @@ pytest_test(
         requirement("jsonschema")
     ],
 )
+
+pytest_test(
+    name = "test_overall_reports",
+    srcs = ["test_overall_reports.py"],
+    deps = [
+        "//:p2p"
```
Comment (limdor) [inline test/BUILD:25]:
```suggestion
        "//reports",
        requirement("pandas"),
```

File: test/BUILD [hunk 2]
```diff
@@ -17,3 +17,19 @@ pytest_test(
...
+    ]
+)
+
+pytest_test(
+    name = "test_diversification_reports",
+    srcs = ["test_diversification_reports.py"],
+    deps = [
+        "//:p2p"
```
Comment (limdor) [inline test/BUILD:33]:
```suggestion
        "//reports",
        requirement("pandas"),
```

File: reports/BUILD [hunk 3]
```diff
@@ -0,0 +1,9 @@
+py_library(
+    name = "reports",
+    srcs = [
+        "__init__.py",
+        "overall.py",
+        "diversification.py"
+    ],
```
Comment (limdor) [inline reports/BUILD:7]: dependencies are missing for pandas and marketplace

**Review by limdor — CHANGES_REQUESTED** [review]
Overall looks good, however some dependencies are missing.
Also the commits should be cleaned up. Are you familiar with git rebase -i? If not we can just have a session about it.

File: test/BUILD (hunk 1 above)
Comment (aquintanas) [reply test/BUILD:25]: I tried before pushing without `requeriment("pandas")" and it didn't work. I will try as you said.

**Comment by aquintanas** [comment]
> Overall looks good, however some dependencies are missing. Also the commits should be cleaned up. Are you familiar with git rebase -i? If not we can just have a session about it.

I will appreciate this opportunity of a master class.

File: reports/BUILD (hunk 3 above)
Comment (aquintanas) [reply reports/BUILD:7]: I'm going to add

File: test/BUILD (hunk 2 above)
Comment (aquintanas) [reply test/BUILD:33]: I'm going to add.

**Comment by aquintanas** [comment]
@limdor, If I add `//reports` in `p2p-lending/BUILD/`:

```
py_library(
    name = "p2p",
    srcs = [
        "__init__.py",
        "logger.py",
        "p2p.py",
    ],
    deps = [
        "//marketplace",
        "//reports",
        requirement("pandas"),
        requirement("xlrd"),
        requirement("openpyxl"),
    ],
    visibility = ["//test:__pkg__"],
)
```

The following error raises:

```
ERROR: /../p2p-lending/BUILD:19:11: no such target '//reports:reports': target 'reports' not declared in package 'reports' defined by /.../p2p-lending/reports/BUILD and referenced by '//:p2p'
```

I think that this dependency should be added, no? If not, why '//marketplace' is a dependency?

**Comment by aquintanas** [comment]
@limdor , I did  `git rabase -i`, can you check if I cleaned commits properly?

**Review by limdor — APPROVED** [review]
Great work!

**Pull request merged**